sdk/core/abort-controller/review/abort-controller.api.md

```diff
@@ -9,7 +9,7 @@ export class AbortController {
     constructor(parentSignals?: AbortSignalLike[]);
     constructor(...parentSignals: AbortSignalLike[]);
     abort(): void;
-    readonly signal: AbortSignal;
+    get signal(): AbortSignal;
```

Comment:
These changes are because the api report is generated with a newer TS version, right?

Reply:
Yes, and these were not committed before in the repo